docs/tutorials/gluon/learning_rate_schedules.md

```diff
@@ -0,0 +1,317 @@
+
+### Manually setting the learning rate: Gluon API only
+
+When using the method above you don't need to manually keep track of iteration count and set the learning rate, so this is the recommended approach for most cases. Sometimes you might want more fine-grained control over setting the learning rate though, so Gluon's `Trainer` provides the `set_learning_rate` method for this.
+
+We replicate the example above, but now keep track of the `iteration_idx`, call the schedule and set the learning rate appropriately using `set_learning_rate`. We also use `schedule.base_lr` to set the initial learning rate for the schedule since we are calling the schedule directly and not using it as part of the `Optimizer`.
+
+
+```python
+net = build_cnn()
+net.collect_params().initialize(mx.init.Xavier(), ctx=ctx)
+
+schedule = mx.lr_scheduler.MultiFactorScheduler(step=steps_iterations, factor=0.1)
+schedule.base_lr = 0.03
+sgd_optimizer = mx.optimizer.SGD()
+trainer = mx.gluon.Trainer(params=net.collect_params(), optimizer=sgd_optimizer)
+
+iteration_idx = 0
+for epoch in range(10):
+    # Iterate through the images and labels in the training data
+    for batch_num, (data, label) in enumerate(train_dataloader):
+        # get the images and labels
+        data = data.as_in_context(ctx)
+        label = label.as_in_context(ctx)
+        # Ask autograd to record the forward pass
+        with mx.autograd.record():
+            # Run the forward pass
+            output = net(data)
+            # Compute the loss
+            loss = softmax_cross_entropy(output, label)
+        # Compute gradients
+        loss.backward()
+        # Update the learning rate
+        lr = schedule(iteration_idx)
+        trainer.set_learning_rate(lr)
+        # Update parameters
+        trainer.step(data.shape[0])
+        # Show loss and learning rate after first iteration of epoch
+        if batch_num == 0:
+            curr_loss = mx.nd.mean(loss).asscalar()
+            curr_lr = trainer.learning_rate
+            print("Epoch: %d; Batch %d; Loss %f; LR %f" % (epoch, batch_num, curr_loss, curr_lr))
+        iteration_idx += 1
+```
+
+Epoch: 0; Batch 0; Loss 2.304286; LR 0.030000 <!--notebook-skip-line-->
+
+Epoch: 1; Batch 0; Loss 0.113910; LR 0.030000 <!--notebook-skip-line-->
+
+Epoch: 2; Batch 0; Loss 0.041778; LR 0.030000 <!--notebook-skip-line-->
+
+Epoch: 3; Batch 0; Loss 0.066455; LR 0.030000 <!--notebook-skip-line-->
+
+Epoch: 4; Batch 0; Loss 0.091915; LR 0.030000 <!--notebook-skip-line-->
+
+Epoch: 5; Batch 0; Loss 0.035965; LR 0.003000 <!--notebook-skip-line-->
+
+Epoch: 6; Batch 0; Loss 0.005169; LR 0.003000 <!--notebook-skip-line-->
+
+Epoch: 7; Batch 0; Loss 0.017087; LR 0.003000 <!--notebook-skip-line-->
+
+Epoch: 8; Batch 0; Loss 0.039928; LR 0.000300 <!--notebook-skip-line-->
+
+Epoch: 9; Batch 0; Loss 0.003349; LR 0.000300 <!--notebook-skip-line-->
```

Comment:
Epoch: 9; Batch 0; Loss 0.003349; LR 0.000300 should be Epoch: 9; Batch 0; Loss 0.003349; LR 0.0000300
I think right?  (i.e. there's a missing 0 in LR).